release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -29,6 +30,20 @@
 
 yaml_path = pathlib.Path("/tmp/ray_v1alpha1_rayservice.yaml")
 
+KILL_WORKER_NODE = "kill_worker_node"
+KILL_HEAD_NODE = "kill_head_node"
+
+
+def generate_cluster_variable():
+    global cluster_id
+    global ray_cluster_name
+    global ray_service_name
+    global locust_id
+    cluster_id = str(uuid.uuid4()).split("-")[0]
+    ray_cluster_name = "cluster-" + cluster_id
+    ray_service_name = "service-" + cluster_id
+    locust_id = "ray-locust-" + cluster_id
+
```

Comment:
Could you print the values for all the `global` variables at the end of this function?

Reply:
We have the config yaml print in the release test. All information can be obtained there :) !